### PR TITLE
Added tests for Location

### PR DESCRIPTION
**Pull request by ethangraham2001**
- Unit tests for `Location` class

### CODE REVIEW

File: app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt
Comment (276746) [inline app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt:1]: line 48 - comment
Is it not 1% margin ?

line 57
equality on float (should not cause problem but still)

line 71-72
same remark as line 57

Comment (saraanej) [reply app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt:1]: You can add your comments directly in the lines when you click on the "+" next to that line. Try it out and let me know if it works for you

File: app/src/main/java/com/github/wanderwise_inc/app/model/location/Location.kt [hunk 1]
```diff
@@ -60,6 +60,7 @@ data class Location(
         // Calculate the angular distance in radians
         val c = 2 * atan2(sqrt(a), sqrt(1 - a))
 
+        println("dist=${earthRadius*c}")
```
Comment (saraanej) [inline app/src/main/java/com/github/wanderwise_inc/app/model/location/Location.kt:63]: It is a good practice to add a comment next to debug lines to remove them easier when not needed :)

File: app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt
Comment (saraanej) [inline app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt:1]: Your tests are good ! You can improve the names of your functions by making them more explicit. We should understand what the test is doing by reading the signature :)

Comment (ethangraham2001) [reply app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt:1]: @276746 indeed unit tests should be placed in test directory, however I was having problems with Jacoco generating code coverage for tests not in the androidTest directory. I'm delaying sorting this issue out for now as it isn't top priority.

Comment (ethangraham2001) [reply app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt:1]: @276746 indeed equality on float isn't best practice, but in this situation it was to test for commutativity and that the result it exactly `0.0` for the distance between a point and itself, hence the goal was to test exact output of the function. I should perhaps have been more explicit in the comments

File: app/src/main/java/com/github/wanderwise_inc/app/model/location/Location.kt (hunk 1 above)
Comment (ethangraham2001) [reply app/src/main/java/com/github/wanderwise_inc/app/model/location/Location.kt:63]: Alright, thank you!

File: app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt
Comment (ethangraham2001) [reply app/src/androidTest/java/com/github/wanderwise_inc/app/model/LocationTest.kt:1]: You're right, looking back at the functions, they're not super clear from their signature. I'll be more explicit going forward

**Review by 276746 — APPROVED** [review]
All good for me

**Pull request merged**